test/unit/busterbunny-raiseEvents-input-validation-tests.js

```diff
@@ -0,0 +1,92 @@
+describe('busterbunny.js raiseEvents', function() {
```

Comment:
Right, because the file name doesn't actually rename the describe text.  My bad.